api/api.yaml

```diff
@@ -130,6 +130,7 @@ paths:
       operationId: set_route_state
       description: >-
         Updates the state of configuration resources on the traffic generator.
+      x-status: deprecated
```

Comment:
x-status has changed to type object. please change as per the model guide.
https://github.com/open-traffic-generator/models/blob/master/MODEL-GUIDE.md